src/components/PromoSheet/index.tsx

```diff
@@ -137,48 +145,65 @@ const PromoSheet = ({
                       </Box>
                     </Box>
                     <Stack alignHorizontal="center" space={{ custom: 13 }}>
-                      <Text
-                        color="secondary60 (Deprecated)"
-                        size="15px / 21px (Deprecated)"
-                        weight="heavy"
-                      >
-                        {lang.t(`promos.${campaignKey}.subheader`)}
+                      <Text color="labelSecondary" size="15pt" weight="heavy">
+                        {subHeader}
+                      </Text>
+                      <Text color="label" size="30pt" weight="heavy">
+                        {header}
                       </Text>
-                      <Heading
-                        color="primary (Deprecated)"
-                        size="30px / 34px (Deprecated)"
-                        weight="heavy"
-                      >
-                        {lang.t(`promos.${campaignKey}.header`)}
-                      </Heading>
                     </Stack>
                   </Box>
                   <Inset horizontal={{ custom: 43.5 }}>
                     <Stack space={isSmallPhone ? '24px' : '36px'}>
-                      <InfoRow
-                        description={lang.t(
-                          `promos.${campaignKey}.info_row_1.description`
-                        )}
-                        gradient={iconGradient}
-                        icon={icon1}
-                        title={lang.t(`promos.${campaignKey}.info_row_1.title`)}
-                      />
-                      <InfoRow
-                        description={lang.t(
-                          `promos.${campaignKey}.info_row_2.description`
-                        )}
-                        gradient={iconGradient}
-                        icon={icon2}
-                        title={lang.t(`promos.${campaignKey}.info_row_2.title`)}
-                      />
-                      <InfoRow
-                        description={lang.t(
-                          `promos.${campaignKey}.info_row_3.description`
-                        )}
-                        gradient={iconGradient}
-                        icon={icon3}
-                        title={lang.t(`promos.${campaignKey}.info_row_3.title`)}
-                      />
+                      {items.map(item => (
+                        <Columns key={item.title} space={{ custom: 13 }}>
+                          <Column width="content">
+                            <MaskedView
+                              maskElement={
+                                <Box
+                                  {...(android && {
```

Comment:
didnt u make a rule to ban prop spreading lol

Reply:
Ah yeah, just a warning for now since we have so many instances of it. I'll have a look and if it's not risky for me to swap this out I will, good call 👍